Formula/libtrace.rb

```diff
@@ -1,14 +1,9 @@
 class Libtrace < Formula
   desc "Library for trace processing supporting multiple inputs"
-  homepage "https://research.wand.net.nz/software/libtrace.php"
-  url "https://research.wand.net.nz/software/libtrace/libtrace-4.0.19.tar.bz2"
-  sha256 "d1a2d756d744c4ffad480f6d4b0bdaca05a9966c87e491992c42007a22317e5a"
-  license "GPL-3.0-or-later"
-
-  livecheck do
-    url :homepage
-    regex(/href=.*?libtrace[._-]v?(\d+(?:\.\d+)+)\.t/i)
-  end
+  homepage "https://github.com/LibtraceTeam/libtrace"
+  url "https://github.com/LibtraceTeam/libtrace/archive/refs/tags/4.0.20-1.tar.gz"
```

Comment:
Is this a fork, or the same developers?

Reply:
yeah, same author, `https://github.com/salcock`